models/chainlink/polygon/chainlink_polygon_ccip_tokens_transferred_logs.sql

```diff
@@ -0,0 +1,59 @@
+{{
+  config(
+    alias='ccip_tokens_transferred_logs',
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+  )
```

Comment:
You need to add the `unique_key` in the config blocks for incremental updates:
```suggestion
    incremental_strategy='merge',
    unique_key=['tx_hash', ...]
  )
```